### PR TITLE
Merge stable to develop

### PR DESCRIPTION
**Pull request by ReimarBauer**
**Purpose of PR?**:

This needs a merge not a squash and merge.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Changelog
 =========
 
+Version 8.3.2
+~~~~~~~~~~~~~
+
+Bug fix release:
+
+All changes:
+https://github.com/Open-MSS/MSS/milestone/100
```
Comment (matrss) [inline CHANGES.rst:10]: Should this include `?closed=1`, as in many previous changelog entries? Also, the milestone is not exhaustive, i.e. the issue/PR for the pint pin is not listed there.

Comment (ReimarBauer) [reply CHANGES.rst:10]: yes, but that was a failure which was not seen on the release creation. this is a merge. we don't do changes in a merge.

Comment (ReimarBauer) [reply CHANGES.rst:10]: that can be fixed when we do a change on docs, or release later.

Comment (matrss) [reply CHANGES.rst:10]: OK, and I just realized that the pint pin change is not part of 8.3.2 but will be in 8.3.3 instead.

**Review by matrss — APPROVED** [review]

File: CHANGES.rst (hunk 1 above)
Comment (ReimarBauer) [reply CHANGES.rst:10]: yes, on conda-forge it is already in the build -1, this is to update our code base. 
develop missed a version string merge.

https://anaconda.org/conda-forge/mss/files

**Pull request merged**